tools/generators/move-packages/index.ts

```diff
@@ -151,7 +152,7 @@ function updateStorybookTypeImport(tree: Tree, schema: MovePackagesGeneratorSche
 function updateCodeOwners(tree: Tree, schema: MovePackagesGeneratorSchema) {
   const { name, destination } = schema;
   if (name) {
-    const pkgName = name.split('/')[1];
+    const pkgName = name.split('/').slice(-1).join('/');
```

Comment:
shouldn't this be the same as on line 173 ? 
`split('/').slice(0, -1).join('/');`

Reply:
ah nevermind :D it's package name

Reply:
I just wanted the normalized package name in this case, changed the approach and used `getProjectConfig` instead to get that